### PR TITLE
Windows compile

### PR DESCRIPTION
**Pull request by redfast00**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,19 +17,24 @@ The resulting PDFs can be found at https://hpcugent.github.io/vsc_user_docs/
...
 
 make all
+```
 
-
-Development
-============
-When making changes, run `./check-version.sh <dir (e.g. intro-HPC)>` to update to new version before making a pr.
+on Windows:
+1. Install MiKTeX and Strawberry Perl
```
Comment (boegel) [inline README.md]: @redfast00 Please include links to the website where these can be downloaded? Also mention that this doesn't require admin rights?

File: README.md [hunk 2]
```diff
@@ -17,19 +17,24 @@ The resulting PDFs can be found at https://hpcugent.github.io/vsc_user_docs/
...
+2. In the MiKTeX console, install  the `latexmk` package
```
Comment (boegel) [inline README.md]: Explain how to start `MiKTeX` console? Install how exactly, does this involve running a command?

File: compile.bat [hunk 3]
```diff
@@ -0,0 +1,13 @@
+@echo off
+SET /p site="Enter site (leuven, antwerpen, gent, brussel): "
+SET /p os="Enter OS (windows, linux, mac): "
+SET /p document="Enter document (intro-Linux, intro-HPC, perfexpert): "
```
Comment (boegel) [inline compile.bat]: Not sure if it's worth the trouble, but do we need to check whether the answer makes sense, i.e. take into account typos?

File: compile.bat [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+
+SET root=%~dp0
+CD %root%%document%
+ECHO %cd%
+
+:forever
+latexmk -pdf -verbose -r ../latexmkrc -jobname="%document%-%os%-%site%" -pdflatex="pdflatex -halt-on-error --file-line-error %%O \"\def\is%os%{1}\def\is%site%{1}\input{%%S}\" " %document%.tex
+pause
+GOTO forever
```
Comment (boegel) [inline compile.bat]: rename `forever` to `build`?

File: README.md (hunk 2 above)
Comment (redfast00) [reply README.md]: No, it's a program named `MiKTeX console`. I don't really know how to work with Windows, but I usually start programs with Win+Q, then type a part of the program name and hit Enter.

**Comment by boegel** [comment]
Looks good to me.

@mselwa Can you test this and let us know whether the `README` updates are clear?

**Comment by mselwa** [comment]
Looks ok and to surprise bat script works even without admin permissions for Leuven :-)

**Comment by boegel** [comment]
Thanks for the feedback @mselwa!

**Pull request merged**